src/stim/simulators/frame_simulator_pybind_test.py

```diff
@@ -216,6 +216,143 @@ def test_set_pauli_flip():
         stim.PauliString('XZ___'),
     ]
 
+def test_broadcast_pauli_errors():
+    sim = stim.FlipSimulator(
+        batch_size=2,
+        num_qubits=3,
+        disable_stabilizer_randomization=True,
+    )
+    sim.broadcast_pauli_errors(
+        pauli='X',
+        mask=np.asarray([
+            [True, False],
+            [False, False],
+            [True, True]]
+        ),
+    )
+    peek = sim.peek_pauli_flips()
+    assert peek == [
+        stim.PauliString("+X_X"),
+        stim.PauliString("+__X")
+    ]
+    sim.broadcast_pauli_errors(
+        pauli='Z',
+        mask=np.asarray([
+            [True, True],
+            [True, False],
+            [False, False]]
+        ),
+    )
+    peek = sim.peek_pauli_flips()
+    assert peek == [
+        stim.PauliString("+YZX"),
+        stim.PauliString("+Z_X")
+    ]
+    sim.broadcast_pauli_errors(
```

Comment:
Add tests with arrays that have fewer and more qubit indices than the current simulator state.

Add tests that array with wrong batch size raise an exception.